models/categories.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var categories = sequelize.define('categories', {
+    name: DataTypes.STRING
+  }, {
+    classMethods: {
```

Comment:
I don't think sequelize is using `classMethods` anymore. If you checkout voting, the association is done outside of the `sequelize.define` parenthesis.